src/dotnet-svcutil/lib/src/Shared/MSBuildProj.cs

```diff
@@ -573,7 +573,7 @@ public bool AddDependency(ProjectDependency dependency, bool copyInternalAssets
                             this.ReferenceGroup.Add(new XElement("Content", new XAttribute("CopyToOutputDirectory", "always"), new XAttribute("Include", Path.Combine(dependency.FullPath.Substring(0, dependency.FullPath.LastIndexOf(Path.DirectorySeparatorChar)), "internalAssets\\**")), new XAttribute("Link", "internalAssets/%(RecursiveDir)%(Filename)%(Extension)")));
                             break;
                         case ProjectDependencyType.Package:
-                            string path = $"$(NuGetPackageRoot){dependency.Name}\\{dependency.Version}\\content\\internalAssets\\**";
+                            string path = $"$(NuGetPackageRoot){dependency.Name}\\{dependency.Version}\\internalAssets\\**";
```

Comment:
Is this needed?  Is it possible to directly reference Nuget package contents?

Reply:
This is needed. We can't directly reference Nuget package as this is used in dotnet-svcutil-lib which targets to netstandard2.0, while the package content multitargets to .NET core and .NET framework.

Reply:
see above comment